services/iplant-groups/src/iplant_groups/routes/domain/attribute.clj

```diff
@@ -94,3 +95,19 @@
 
 (s/defschema PermissionAllowed
   {:allowed (describe Boolean "Whether this permission should be marked as allowed or disallowed (latter to override an inherited permission).")})
+
```

Comment:
The remaining changes in this file define the schema for the `PUT /attributes/:attribute-name/permissions` endpoint.